src/reference/sectionA_definitions.inc

```diff
@@ -188,23 +188,11 @@ The following data representation formats are defined for use in this specificat
 
    .. container:: issue-data-repr-identifier-unicode
 
-      1. SHALL be a sequence of Unicode characters.
-
-   .. container:: issue-data-repr-identifier-latin-alphanum
-
-      2. SHALL NOT contain any characters except :ref:`Basic Latin alphanumeric characters<specA_basic_latin_alphanumeric_character>` and :ref:`Basic Latin underscores<specA_basic_latin_underscore>`.
-
-   .. container:: issue-data-repr-identifier-at-least-one-alphanum
-
-      3. SHALL contain at least one :ref:`alphabetic<specA_basic_latin_alphabetic_character>` character.
-
-   .. container:: issue-data-repr-identifier-begin-euro-num
-
-      4. SHALL NOT begin with a :ref:`numeral<specA_european_numeral>` or an :ref:`underscore<specA_basic_latin_underscore>`.
+      1. SHALL consist of a single :ref:`Basic Latin alphabetic character<specA_basic_latin_alphabetic_character>`, which MAY be followed by any combination of :ref:`Basic Latin alphanumeric characters<specA_basic_latin_alphanumeric_character>` and :ref:`Basic Latin underscores<specA_basic_latin_underscore>`.
```

Comment:
Perhaps: "SHALL **begin** with a single ... which MAY be followed "... etc?  It doesn't change the meaning, but it makes it easier (I think ...) to get the correct idea the first time you read it?  

Also, maybe "Latin alphanumeric characters **and/or** Basic Latin underscores" ?

Reply:
I'm gonna stick with "consist of" because if we say "begins with X, may be followed by Y", then we're not explicitly saying it isn't then followed by P,Q,R and/or S